### PR TITLE
915 - Apply new FSPIOP error handling to email notifier

### PR DESCRIPTION
**Pull request by gibaros**
-remove package-lock.json from .gitignore
-update package.json depdendencies and scripts
-update existing error handling with FSPIOPError framework
-update unit tests as required by update to existing error handling with FSPIOPError framework

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,14 +6,14 @@
   "dependencies": {
```
Comment (rmothilal) [inline package.json]: Please update version of package.json and run npm install

File: package.json [hunk 2]
```diff
@@ -48,20 +48,20 @@
   "homepage": "https://github.com/mojaloop/email-notifier#readme",
   "devDependencies": {
     "chai": "4.2.0",
-    "coveralls": "3.0.5",
+    "coveralls": "3.0.6",
     "faucet": "0.0.1",
-    "istanbul": "1.1.0-alpha.1",
+    "istanbul": "0.4.5",
```
Comment (rmothilal) [inline package.json]: please revert version of istanbul to 1.1.0-alpha.1

File: package.json [hunk 3]
```diff
@@ -48,20 +48,20 @@
...
     "jsonpath": "1.0.2",
     "mocha": "6.2.0",
```
Comment (rmothilal) [inline package.json]: Dont believe your tests require mocha and chai. Please remove

File: package.json [hunk 4]
```diff
@@ -48,20 +48,20 @@
   "homepage": "https://github.com/mojaloop/email-notifier#readme",
   "devDependencies": {
     "chai": "4.2.0",
```
Comment (rmothilal) [inline package.json]: Dont believe your tests require mocha and chai

Comment (rmothilal) [reply package.json]: Please move tape and tapes from dependencies to dev dependencies

File: package.json (hunk 1 above)
Comment (gibaros) [reply package.json]: Performed per review.

File: package.json (hunk 2 above)
Comment (gibaros) [reply package.json]: Performed per review.

File: package.json (hunk 3 above)
Comment (gibaros) [reply package.json]: Performed per review.

File: package.json (hunk 4 above)
Comment (gibaros) [reply package.json]: Performed per review.

**Pull request merged**